@navikt/core/react/src/date/utils/locale.ts

```diff
@@ -29,3 +29,19 @@ export const getTranslations = (locale: string | undefined) => {
       return undefined;
   }
 };
+
+/**
+ * @private
+ * Temporary for backwards compatibility with locale prop
```

Comment:
For temp-comments, it could be nice to have a small comment describing the requirements for it being removed or "not" temporarily anymore if that makes sense 🤔